platform-sdk/swirlds-logging/src/test/java/com/swirlds/logging/api/internal/LoggingSystemTest.java

```diff
@@ -785,6 +803,28 @@ public void accept(LogEvent logEvent) {
         testSpec(loggingSystem);
     }
 
+    @Test
+    @DisplayName("Test that installing multiple known handler type and enabled throws no exception")
+    void testAddMultipleFileHandler() {
+        // given
+        final Configuration configuration = new TestConfigBuilder()
+                .withValue("logging.handler.CRYPTO_FILE.enabled", "true")
+                .withValue("logging.handler.CRYPTO_FILE.type", "file")
+                .withValue("logging.handler.CRYPTO_FILE.file", "crypto.log")
+                .withValue("logging.handler.TRANSACTION_FILE.enabled", "true")
+                .withValue("logging.handler.TRANSACTION_FILE.type", "file")
+                .withValue("logging.handler.TRANSACTION_FILE.file", "transaction.log")
+                .getOrCreateConfig();
+
+        loggingSystem = new LoggingSystem(configuration);
+
+        // when
+        loggingSystem.installHandlers();
+
+        // then
+        assertThat(loggingSystem.getHandlers()).hasSize(2);
+    }
```

Comment:
Let's create a test that logging really ends in a file ;)

Reply:
Done